packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -28,7 +28,8 @@ class AndroidWorkflow extends Workflow {
     androidValidator.addValidator(new Validator(
       'Android SDK',
```

Comment:
Perhaps in the download instructions? This check will pass if we can locate an Android SDK.